app/views/public/planning_guides/elevations/proposed.html.erb

```diff
@@ -27,7 +27,7 @@
           <li><strong>Be clearly labelled</strong></li>
           <li><strong>Show all sides of the proposal </strong></li>
         </ul>
-        <h2>You should include</h2>
+        <h2 class="govuk-heading-m">You should include</h2>
         <ul>
```

Comment:
Maybe in another PR but shouldn't these lists have `govuk-*` styles on them?

Reply:
Oh yes, hadn't spotted! I'll get this merged and do another sweep for other elements missing the styling, and that can be another PR.